config.go

```diff
@@ -289,6 +293,8 @@ type Config struct {
 	MaxChanSize                   int64         `long:"maxchansize" description:"The largest channel size (in satoshis) that we should accept. Incoming channels larger than this will be rejected"`
 	CoopCloseTargetConfs          uint32        `long:"coop-close-target-confs" description:"The target number of blocks that a cooperative channel close transaction should confirm in. This is used to estimate the fee to use as the lower bound during fee negotiation for the channel closure."`
 
+	CommitTime time.Duration `long:"commit-time" description:"The maximum time that is allowed to pass between receiving a channel state update and signing the next commitment. Setting this to a longer duration allows for more efficient channel operations at the cost of latency."`
+
```

Comment:
nit: instead of "more efficient channel operations" you could write "larger channel throughput" as that may be more descriptive for this parameter.

Reply:
I considered that, but as it is currently, increasing this value doesn't yet increase throughput dramatically. There are just too many non-batched txes going one. I felt that 'larger channel throughput' would overpromise a bit, even though throughput should indeed be slightly larger.